### DISCUSSION TITLE
Format single stat panel with thousands separator

### DISCUSSION THREAD

**Discussion by brylie** [grafana/grafana#36994] [Dashboards, Panels & Visualizations]
Creating a feature request, as [recommended in community discussion](https://community.grafana.com/t/how-to-format-single-stat-panel-to-add-comma-separators/6035).

Currently, the **single stat** widget displays numbers in their 'raw' form:

![screenshot from 2018-03-09 13-28-25](https://user-images.githubusercontent.com/17307/37274825-3f38e78c-25e6-11e8-865b-3e9cb6157481.png)

# Feature request
Allow numbers to be formatted to make them easier to read, such as by adding thousands separators.

**Comment by pixelsstudios** [comment]
+1 for this!

**NOTE:** This is **not** a currency or internationalisation issue, as it has previously been confused for. It's changing 29286 to 29,286

**Comment by brylie** [comment]
> This is not a[n] internationalisation issue

Although, it is worth considering different [methods for digit grouping](https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping). E.g. it may be desirable to let the user select the thousands separator from a list.

**Comment by dbrgn** [comment]
It seems that under "Field > Standard Options > Unit" you can set the unit to "Misc > Locale format".

![2021-01-12-144227_887x240_scrot](https://user-images.githubusercontent.com/105168/104321812-67761280-54e4-11eb-8b9e-efa76a3313f6.png)

**Reply by JEEVAN28** [reply]
In India we separate above value 77,62,990
and even larger number as below

77,77,77,62,990

is this possible

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
